compat-data-schema.md

```diff
@@ -0,0 +1,365 @@
+# The browser-compat-data JSON format
```

Comment:
I created issue #177 for the copy-edit part.
I updated the README file to link to the compat-data-schema.md instead of the description of the old schema.